triggers/daml/BUILD.bazel

```diff
@@ -13,6 +13,8 @@ genrule(
       TMP_DIR=$$(mktemp -d)
       mkdir -p $$TMP_DIR/daml/Daml/Trigger
       cp -L $(location Daml/Trigger.daml) $$TMP_DIR/daml/Daml
+      cp -L $(location Daml/Trigger/Assert.daml) $$TMP_DIR/daml/Daml/Trigger
+      cp -L $(location Daml/Trigger/Internal.daml) $$TMP_DIR/daml/Daml/Trigger
       cp -L $(location Daml/Trigger/LowLevel.daml) $$TMP_DIR/daml/Daml/Trigger
```

Comment:
Should we set `exposed-modules` to hide `Daml/Trigger/Internal.daml`.

Reply:
Good idea, done.

Reply:
`Daml.Trigger.Internal` is exposed in the docs nonetheless. I've opened https://github.com/digital-asset/daml/issues/4268 to track this.